ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -44,9 +46,17 @@ azure_connection::list_objects(std::vector<std::string> &objects) const
     return 0;
 }
 
+/*
+ * Puts an object given a reference to the filename
+ */
 int
 azure_connection::put_object(const std::string &file_name) const
 {
+    std::filesystem::path p(file_name);
+    if (!std::filesystem::exists(p))
+        throw std::invalid_argument(file_name + ": No such file exists.");
+    auto blobClient = _azure_client.GetBlockBlobClient(p.filename());
```

Comment:
pls change blobClient to blob_client to abide by cpp conventions

Reply:
also PR title and jira ticket title should update putObect to put_object

Reply:
addressed in https://github.com/wiredtiger/wiredtiger/pull/8718/commits/7685d04439a42c0c444b046c8fcd8744cb9ad9db